examples/demo/site/modules/auth/mail_templates/admin_account_evaluation.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Account Evaluation</title>
+  <meta name="description" content="Account Evaluation">
+  <meta name="author" content="$sitename">
+</head>
+
+<body>
+    <h2> Account Evaluation </h2>
+	<p>The user $user ($email) wants to register to the site <a href="$host">$sitename</a></p>
+
+	<blockquote><p>This is his/her application.</p>
```

Comment:
"This is his/her application."

what is this is a bot ... we should have  "his/her/its" .. and if this is a group of people, we should have "his/her/its/their", and even "ours" in case the admin is part of this group ;)

more seriously, we could have

"Application" or "Motivation" or "User application" ... or similar